iliad/.htaccess

```
@@ -19,33 +19,74 @@ RewriteRule ^oim$ https://github.com/ILIAD-ocean-twin/oim [R=302,L]
 
 ### API
 # Base
-RewriteRule ^api$ https://grlc.io/api-git/ILIAD-ocean-twin/api/base [R=302,L]
-RewriteRule ^api/v1.0$ https://grlc.io/api-git/ILIAD-ocean-twin/api/base [R=302,L]
-RewriteRule ^api/v1.0/$ https://grlc.io/api-git/ILIAD-ocean-twin/api/base [R=302,L]
+RewriteRule ^api$ https://dev.grlc.io/api-git/ILIAD-ocean-twin/JF-API/base [R=302,L]
+RewriteRule ^api/v1.0$ https://dev.grlc.io/api-git/ILIAD-ocean-twin/JF-API/base [R=302,L]
+RewriteRule ^api/v1.0/$ https://dev.grlc.io/api-git/ILIAD-ocean-twin/JF-API/base [R=302,L]
```

You can simplify the optional trailing slash as `/?`.
```suggestion
RewriteRule ^api/v1.0/?$ https://dev.grlc.io/api-git/ILIAD-ocean-twin/JF-API/base [R=302,L]
```